requirements/bench.txt

```diff
@@ -1,4 +1,8 @@
 requests>=2.21.0
+smac[all]>=0.13.1
+scipy=1.6  # 1.6 required for SMAC because deprecated usage of sobol_seq
```

Comment:
this is going to be a huge pain point as I expect some other packages will want other versions